src/ucp/tag/rndv.c

```diff
@@ -28,6 +28,34 @@ static int ucp_tag_rndv_is_get_op_possible(ucp_ep_h ep, ucp_lane_index_t lane,
     }
 }
 
+static int ucp_tag_rndv_is_mrail_get_op_possible(ucp_ep_h ep, ucp_request_t *req)
```

Comment:
can we generalize the logic, so it will always be considered "multi-rail", and rndv_max_lanes=1 will just be one of the possible cases? so this function will not be needed?